src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -11,11 +11,14 @@ namespace OpenTelemetry.Metrics;
 
 internal sealed class AggregatorStore
 {
+    internal readonly HashSet<string>? TagKeysInteresting;
     internal readonly bool OutputDelta;
     internal readonly bool OutputDeltaWithUnusedMetricPointReclaimEnabled;
     internal readonly int CardinalityLimit;
     internal readonly bool EmitOverflowAttribute;
     internal readonly ConcurrentDictionary<Tags, LookupData>? TagsToMetricPointIndexDictionaryDelta;
+    internal readonly ExemplarSamplingHelper ExemplarSampler;
```

Comment:
i understand we will remove this in the future, but to make it easier - could you name this ExemplarFilteringHelper?

Reply:
How's this? https://github.com/open-telemetry/opentelemetry-dotnet/pull/5364/commits/fb9b07ad25c552605eb2a1a065429d32bbd679e6